Server/Main.cs

```diff
@@ -87,6 +87,8 @@ public static TimeSpan ProfileTime
 		}
 
 		public static bool Service { get; private set; }
+
+        public static bool NoConsole { get; private set; }

```

Comment:
I would propose to change the name of this property to `NoInteraction`. IMHO it reflects better the intention of this property, which is to avoid querying the user through the console for input during execution.